libs/components/src/wizard/stories/vl-wizard-pane.stories-doc.mdx

```diff
@@ -0,0 +1,48 @@
+import { ArgTypes, Canvas } from '@storybook/blocks';
+import * as VlWizardPaneStories from './vl-wizard-pane.stories';
+
+# Wizard Pane
+
+Gebruik de wizard pane om een stap in de wizard voor te stellen.
```

Comment:
'Gebruik de `wizard pane` component om een stap in de wizard af te beelden.'

Reply:
✅